### PR TITLE
Service list view doc update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/apm/traces/services-list-map.md [hunk 1]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
...
-The **Services List** view provides a compact, high-level overview of your application service health insights and KPIs. You can filter and sort your KPI data using the filters and column headers.
+The **Services List** view provides a compact, high-level overview of your application service health insights, Alerts, and KPIs. You can filter and sort your KPI data using the filters and column headers.
 
 ![services-list-display](/img/traces/services-list-filters.png)
 
 KPIs listed in the table reflect the previous 15 minutes of data. Clicking on any service will open the Entities panel allowing for further drill-down.
 
+### Alerts
+
+The **Alert status** column will display the most severe alert status along with the total number of alerts associated with a given service. The **Status** column will display the services with alerts, to get detailed information on the alerts, click on the associated service to open the entity inspector.
```
Comment (jpipkin1) [inline docs/apm/traces/services-list-map.md:25]:
```suggestion
The **Alert status** column will display the most severe alert status along with the total number of alerts associated with a given service. The **Status** column will display the services with alerts. To get detailed information on the alerts, click on the associated service to open the entity inspector.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I just made one tiny correction.

File: docs/apm/traces/services-list-map.md [hunk 2]
```diff
@@ -95,6 +116,9 @@ Manual anomaly detection allows you to specify different detection settings for
 
 <img src={useBaseUrl('img/traces/services-list-map-manual-detection-settings.png')} alt="services manual anomaly detection settings" width="400"/>
 
+### Alerts
+
+**Alerts** allows you to consider all associated alerts, which are currently active or resolved within the selected time.
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:121]:
```
If this option is enabled, for a given service its status will be influenced by the presence of active alerts (or those, which have been resolved within the selected time range) associated with the service.
```
WDYT?

File: docs/apm/traces/services-list-map.md [hunk 3]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
 
 ## Services List view
 
-The **Services List** view provides a compact, high-level overview of your application service health insights and KPIs. You can filter and sort your KPI data using the filters and column headers.
+The **Services List** view provides a compact, high-level overview of your application service health insights, Alerts, and KPIs. You can filter and sort your KPI data using the filters and column headers.
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:17]: `Alerts` -> `alerts`?

File: docs/apm/traces/services-list-map.md [hunk 4]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
...
 
 ![services-list-display](/img/traces/services-list-filters.png)
 
 KPIs listed in the table reflect the previous 15 minutes of data. Clicking on any service will open the Entities panel allowing for further drill-down.
 
+### Alerts
+
+The **Alert status** column will display the most severe alert status along with the total number of alerts associated with a given service. The **Status** column will display the services with alerts. To get detailed information on the alerts, click on the associated service to open the entity inspector.
+
+**Alerts** in the Service List are displayed based on the configured [Monitors](/docs/alerts/monitors/overview/) in the Sumo Logic platform for a given host.
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:27]: `for a given host` -> `for a given service`

File: docs/apm/traces/services-list-map.md [hunk 5]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
...
+To view alert details, click on a row and view it in the right-hand-side panel.
+
+In addition, you can narrow down the list of visible services by the type of active alerts for a given service. Service List can filter based on the following alert statuses:
+
+- Critical
+- Warning
+- Missing Data
+- Normal
+
+A Service is displayed in the list if at least one of the alerts is in the selected state.
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:40]: `Service` -> `service`?

File: docs/apm/traces/services-list-map.md [hunk 6]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
...
-The **Services List** view provides a compact, high-level overview of your application service health insights and KPIs. You can filter and sort your KPI data using the filters and column headers.
+The **Services List** view provides a compact, high-level overview of your application service health insights, Alerts, and KPIs. You can filter and sort your KPI data using the filters and column headers.
 
 ![services-list-display](/img/traces/services-list-filters.png)
 
 KPIs listed in the table reflect the previous 15 minutes of data. Clicking on any service will open the Entities panel allowing for further drill-down.
 
+### Alerts
+
+The **Alert status** column will display the most severe alert status along with the total number of alerts associated with a given service. The **Status** column will display the services with alerts. To get detailed information on the alerts, click on the associated service to open the entity inspector.
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:25]: I would delete this line...

File: docs/apm/traces/services-list-map.md [hunk 7]
```diff
@@ -14,12 +14,33 @@ To open the Services List and Map, click **+ New** > **Services**.
...
+
+**Alerts** in the Service List are displayed based on the configured [Monitors](/docs/alerts/monitors/overview/) in the Sumo Logic platform for a given host. 
+
+<img src={useBaseUrl('/img/observability/alerts_host_list.png')} alt="alerts_host_list" style={{border: '1px solid black'}} width="400"/>
+
```
Comment (njoy89) [inline docs/apm/traces/services-list-map.md:30]: ... and add here:
```
The **Alert status** column displays the most severe alert status along with the total number of alerts associated with a given service.
```

File: docs/apm/traces/services-list-map.md (hunk 3 above)
Comment (JV0812) [reply docs/apm/traces/services-list-map.md:17]: it is alerts, and I have fixed that.

File: docs/apm/traces/services-list-map.md (hunk 2 above)
Comment (JV0812) [reply docs/apm/traces/services-list-map.md:121]: Even this looks good to me. I will add this.

**Review by kimsauce — APPROVED** [review]

File: docs/apm/traces/services-list-map.md [hunk 8]
```diff
@@ -86,15 +100,20 @@ An anomaly is a spike in latency and errors or a dip in requests. Here's how to
...
 
 ### Manual
 Manual anomaly detection allows you to specify different detection settings for each KPI. Click the toggle switch to manually enable or disable each KPI highlight as desired.
 
-<img src={useBaseUrl('img/traces/services-list-map-manual-detection-settings.png')} alt="services manual anomaly detection settings" width="400"/>
+<img src={useBaseUrl('img/traces/services-list-map-manual-detection-settings.png')} style={{border: '1px solid black'}} alt="services manual anomaly detection settings" width="400"/>
+
+### Alerts
+
+If **Alerts** option is enabled, for a given service its status will be influenced by the presence of active alerts (or those, which have been resolved within the selected time range) associated with the service.
```
Comment (kimsauce) [inline docs/apm/traces/services-list-map.md:114]:
```suggestion
If the **Alerts** option is enabled for a given service, its status will be influenced by the presence of active alerts (or those, which have been resolved within the selected time range) associated with the service.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

**Pull request merged**